modules/sql-engine/src/main/codegen/config.fmpp

```diff
@@ -53,6 +56,7 @@ data: {
 #     "KEY_TYPE" // already presented in Calcite
       "TREE"
       "HASH"
+#      "ZONE"
```

Comment:
why it commented?

Reply:
Thanks, I forgot to remove it from this list, It is not reserved keyword in Calcite.